scripts/build

```diff
@@ -53,3 +53,9 @@ fi
 if ! [ "$LOCAL" = "false" ]; then
     build-binaries ${OS} ${ARCH}
 fi
+
+# When only running "dapper build", you may have to copy the binaries to "dist".
+if [ "$COPY_TO_DIST" = "true" ]; then
```

Comment:
Can this be the default behavior? When you run dapper build it is always going to cp to dist/artifacts

Reply:
Sure thing, but we should remove it from `scripts/package` then. Thoughts?

Reply:
yep, I think so

Reply:
Ok great, will do